Xamarin.Essentials/FilePicker/FilePicker.android.cs

```diff
@@ -14,9 +14,7 @@ public static partial class FilePicker
     {
         static async Task<FilePickerResult> PlatformPickFileAsync(PickOptions options)
         {
-            // we only need the permission when accessing the file, but it's more natural
-            // to ask the user first, then show the picker.
-            await Permissions.RequireAsync(PermissionType.ReadExternalStorage);
+            await Permissions.RequestAsync<Permissions.StorageRead>();
```

Comment:
Why remove the comment? I think that is still useful for future viewers.

Reply:
I've added it back